docs/pages/database-access/guides/clickhouse-self-hosted.mdx

```diff
@@ -121,95 +111,149 @@ you're running multiple services add `--output=stdout` to print the config in
 your terminal, and manually adjust `/etc/teleport.yaml`.
 </Admonition>
 
+{/* Inlining docs/pages/includes/database-access/self-hosted-config-start.mdx
+so we can use the Var component for the protocol.*/}
+
 On the host where you will run the Teleport Database Service, start Teleport
 with the appropriate configuration.
 
-Generate a configuration file at `/etc/teleport.yaml` for the Database Service:
+Note that a single Teleport process can run multiple different services, for
+example multiple Database Service agents as well as the SSH Service or Application
+Service. The step below will overwrite an existing configuration file, so if
+you're running multiple services add `--output=stdout` to print the config in
+your terminal, and manually adjust `/etc/teleport.yaml`.
 
-<Tabs>
-<TabItem label="HTTP">
+Run the following command to generate a configuration file at
+`/etc/teleport.yaml` for the Database Service. Update 
+<Var name="example.teleport.sh" /> to use the host and port of the Teleport Proxy
+Service:
 
 ```code
-$ teleport db configure create \
+$ sudo teleport db configure create \
```

Comment:
Why `sudo`?

Reply:
I don't think there was a conscious reason. My guess is I copied this command from another guide that used it, `sudo` included.

Reply:
```bash
$ teleport db configure create -o file
ERROR: could not write to config file, missing sudo?
	open /etc/teleport.yaml: permission denied
```

this must be why 😅 please re-add